### DISCUSSION TITLE
Nonmonotonic time tags

### DISCUSSION THREAD

**Discussion by Sudo-Rob** [weather-gov/api#555] [Q&A] [answered]
I'm observing time tag 'stutter' in the ordering of forecast data (presently monitoring KBWI).  These occur around two or three times throughout the day.  What I'm seeing is time tags decrement and recover.  These can happen at any time, but they are more impactful when occurring at the local midnight transition.  

Here is an example of what I'm seeing:

![image](https://user-images.githubusercontent.com/56487711/171664765-62d25010-c329-4d3a-84af-f093c1d7e889.png)

It seems to be a different signature than [#492](https://github.com/weather-gov/api/discussions/492).

**Comment by jah1007** [comment, marked as answer]
Within the response payload, which parameter are you reading to get these time values?

**Reply by Sudo-Rob** [reply]
It's ``periods: 0: StartTime``
